app/javascript/src/components/Dashboard/Notes/MenuBar.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+
+import { Search, Settings, Plus } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar as NeetoMenuBar } from "neetoui/layouts";
+
+import { MENUBAR_BLOCKS } from "./constants";
+
+const MenuBar = () => {
+  const [isSearchCollapsed, setIsSearchCollapsed] = useState(true);
+  return (
```

Comment:
Add a newline before statements that exit the scope. It's similar to [this pattern](https://academy.bigbinary.com/learn-react/pages/best-practices-associated-with-conditional-statements#:~:text=Add%20a%20blank%20line%20after%20each%20guard%20clause%20to%20explicitly%20show%20that%20the%20statement%20exits%20the%20function%20scope.).